akvo/rsr/models/report.py

```diff
@@ -39,6 +39,7 @@ class Report(models.Model):
         help_text=_(u'Enter the parametrized path for downloading the report'),
     )
     formats = models.ManyToManyField(ReportFormat)
+    organisations = models.ManyToManyField('Organisation')
```

Comment:
@zzgvh I don't think a ManyToManyField needs `null=True` or `blank=True`. I couldn't find a reference in the official documentation, but I found this [SO answer](https://stackoverflow.com/a/18244527/1225068), and it makes sense to me.